### DISCUSSION TITLE
catalog访问hive，use database或show tables太慢

### DISCUSSION THREAD

**Discussion by EthanLiuGTI** [apache/doris#37617] [A - General / Q&A]
通过catalog访问hive，执行use database或show tables时，doris都要重新去ranger拉取策略，并没有使用缓存，由于生产上面表和策略特别多，导致需要几分钟才能执行成功，甚至长时间无法执行成功

**Comment by ixzc** [comment]
中文问题在ask.selectdb.com提个帖子

**Comment by gagan211** [comment]
@EthanLiuGTI 
can you tell me about the steps that you used to install apache ranger and integrate it with doris as i am not able to connect the 2 of them.
